config/deploy/old_prod.rb

```diff
@@ -1,5 +1,8 @@
 server 'sul-lyberservices-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
+set :repo_url, 'https://github.com/sul-dlss/pre-assembly.git'
```

Comment:
when the repo_url was set to use ssh:// protocol not used in deploy.rb, it didn't work for me  .... is there a reason it was changed?